index.html

```diff
@@ -92,8 +94,154 @@ <h2 class="card__question">Woher kommt der Name "Frisbee"?</h2>
+        <h2 class="card__question">
+          Wofür ist die Inselgruppe Tuvalu im Südpazifik berühmt?
+        </h2>
+        <button class="card__button">Antwort</button>
+        <p class="card__answer card__answer--hidden">
+          Die aus neun Korallen-Atollen bestehende Inselgruppe mit ihren 11.000
+          Einwohnern ist durch ihre Internet-Domain ".tv" berühmt.
+        </p>
+        <ul class="tags">
+          <li class="tags__item">Internet</li>
+          <li class="tags__item">Geografie</li>
+          <li class="tags__item">Pazifik</li>
+          <li class="tags__item">Insel</li>
+        </ul>
+      </section>
+      <section class="card">
+        <button
+          class="card__bookmark"
+          aria-label="Quiz-Frage als Favorit markieren"
+        ></button>
+        <h2 class="card__question">
+          Wie schwer war das erste in Serie produzierte Handy?
+        </h2>
+        <button class="card__button">Antwort</button>
+        <p class="card__answer card__answer--hidden">
+          Das erste Serienhandy der Welt, das Motorola Dynatac (auch "Knochen"
+          genannt), wog 800 Gramm. So viel wie acht Tafeln Schokolade.
+        </p>
+        <ul class="tags">
+          <li class="tags__item">Technik</li>
+          <li class="tags__item">Digital</li>
+          <li class="tags__item">Handy</li>
+          <li class="tags__item">Gewicht</li>
+        </ul>
+      </section>
+      <section class="card">
+        <button
+          class="card__bookmark"
+          aria-label="Quiz-Frage als Favorit markieren"
+        ></button>
+        <h2 class="card__question">Woher kommt der Name "Frisbee"?</h2>
+        <button class="card__button">Antwort</button>
+        <p class="card__answer card__answer--hidden">
+          Der Name geht auf die Bäckerei "Frisbie" in Conneticut zurück. Deren
+          kreisrunde Backformen aus Zinn waren die Vorläufer der fliegenden
+          Plastikscheiben.
+        </p>
+        <ul class="tags">
+          <li class="tags__item">Sport</li>
+          <li class="tags__item">Freizeit</li>
+          <li class="tags__item">Name</li>
+          <li class="tags__item">Geschichte</li>
+        </ul>
+      </section>
+    </main>
+
+    <main class="main main--hidden" id="create">
+      <section class="card__create">
+        <form>
+          <br />
+          <label for="question">Deine Frage</label><br />
+          <textarea id="question" name="question" rows="3" cols="32">
+Hier deine Frage stellen
+          </textarea>
+          <br />
+          <label for="answer">Deine Antwort</label><br />
+          <textarea id="answer" name="answer" rows="3" cols="32">
+Hier deine Antwort geben
+          </textarea>
+          <br />
+          <label for="tags">Schlagworte</label><br />
+          <textarea id="tags" name="tags" rows="3" cols="32">
+Tags durch Komma trennen
+          </textarea>
+          <br />
+          <input type="submit" value="Submit" />
```

Comment:
It's personal preference, but I usually prefer using a `<button>Submit</button>` in a form as a submit button 🙂 But your solution is correct as well!

Reply:
I see, gonna change that, too.